plugins/modules/elastic_agentpolicy_info.py

```diff
@@ -37,7 +65,8 @@ def main():
         password=dict(type='str', no_log=True, required=True),   
         verify_ssl_cert=dict(type='bool', default=True),
         agent_policy_name=dict(type='str'),
-        agent_policy_id=dict(type='str')
+        agent_policy_id=dict(type='str'),
+        deployment_info=dict(type='dict', default=None)
```

Comment:
we should probably look at more tightly-specifying the `deployment_info` object using `options`.  not all that important for us, but will make the code more usable for others later